src/main/java/com/thebois/models/world/resources/Tree.java

```diff
@@ -0,0 +1,43 @@
+package com.thebois.models.world.resources;
+
+import com.thebois.models.Position;
+
+/**
+ * Resource of type tree.
+ */
+public class Tree extends AbstractResource {
+
+    /**
+     * Instantiate a new tree resource at given position.
+     *
+     * @param position The position where the tree resource should be created.
+     */
+    public Tree(final Position position) {
+        super(position);
+    }
+
+    /**
+     * Instantiate a new tree resource at given position.
+     *
+     * @param posX X coordinate for the tree resource.
+     * @param posY Y coordinate for the tree resource.
+     */
+    public Tree(final float posX, final float posY) {
+        super(posX, posY);
```

Comment:
Should probably call `this` instead of `super`. In case `Tree` needs custom constructor instantiation.